modules/node_modules/@ncigdc/dux/cart.js

```diff
@@ -136,6 +137,39 @@ function fetchFiles (currentFilters, total) {
   };
 }
 
+function fetchFilesAndRemove(currentFilters: ?Object, size: number): Function {
+  return async (dispatch, getState) => {
+    const existingFiles = getState().cart.files;
+
+    if (!existingFiles.length) {
+      dispatch(notify({
+        action: 'full',
```

Comment:
should probably rename this `action` property, took me a while to figure out where it's used because it's not a redux action, it's just a key into a styles object here https://github.com/NCI-GDC/portal-ui/blob/relay/modules/node_modules/%40ncigdc/uikit/Notification.js#L69 where 'add' has a green background and 'remove' has a red background. So this should be `action: 'remove'`?